core/src/main/scala/pureconfig/ConfigCursor.scala

```diff
@@ -82,6 +82,21 @@ sealed trait ConfigCursor {
       case v => Left(CannotConvert(v.toString, "Long", "Unable to convert Number to Long"))
     })
 
+  /**
+   * Casts this cursor to a long representing a memory size according to the way com.typesafe.config.Config#getBytes(java.lang.String) does.
+   *
+   * @return a `Right` with the memory size value pointed to by this cursor if the cast can be done, `Left` with a list of
+   *         failures otherwise.
+   */
+  def asBytes: ConfigReader.Result[Long] = scopeFailure {
```

Comment:
@jcazevedo  coming to think of it, do u think it's more appropriate to name this: `asMemorySize`, current name seems like the plural of the `asByte` method